src/components/NFTFilters/NFTFilters.spec.tsx

```diff
@@ -28,22 +32,78 @@ describe.each([
   })
 })
 
+describe(`when the category is ${MainCategory.WEARABLE}`, () => {
+  let screen: RenderResult
+
+  beforeEach(() => {
+    screen = renderNftFilters({ filters: { category: MainCategory.WEARABLE } })
+  })
+
+  it('should render the smart wearable filter', () => {
+    expect(screen.getByTestId(SMART_WEARABLE_FILTER_DATA_TEST_ID)).toBeInTheDocument()
+  })
+})
+
 describe('when changing rarity', () => {
+  let onChange: jest.Mock
+  let screen: RenderResult
+
+  beforeEach(() => {
+    onChange = jest.fn()
+    screen = renderNftFilters({ onChange })
+  })
+
   it('should call onChange function with the right params', async () => {
```

Comment:
It's not part of the changes from the PR but we should enforce not using this kind of descriptions for the tests.

Instead of `with the right params`, use `with the common item rarity`

Reply:
https://github.com/decentraland/profile/pull/275/files#diff-37ab30664bf95c3f6b4dbb8eed07518004cc094b7dc1e9531a6bc54a504f487cR88 <- Like here ❤️